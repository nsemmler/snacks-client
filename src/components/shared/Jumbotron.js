import {Jumbotron} from 'reactstrap';
import React from 'react'

const Jumbo = ({props}) => {
  return (<Jumbotron style={props.height}>
    <div className="img"></div>
    <div className="container">
      <h1 className="display-3">{props.title}</h1>
      <hr className="my-2"/>
      <p className="lead">
        {props.subtitle}
      </p>
    </div>
  </Jumbotron>)
}

export default Jumbo
