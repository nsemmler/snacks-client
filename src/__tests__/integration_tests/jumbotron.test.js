import React from 'react'
import renderer from 'react-test-renderer'
import { MemoryRouter } from 'react-router-dom'
import Jumbotron from '../../components/shared/Jumbotron'

describe('Jumbotron Component', () => {
  test('renders consistently', () => {
    const sample = {
      height: '100px',
      title: 'Title',
      subtitle: 'Subtitle'
    }
    const form = renderer.create(
      <MemoryRouter>
        <Jumbotron props={ sample }/>
      </MemoryRouter>
    ).toJSON()
    expect(form).toMatchSnapshot()
  })
})
