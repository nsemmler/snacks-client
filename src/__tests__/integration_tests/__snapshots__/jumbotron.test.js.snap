// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Component renders consistently 1`] = `
<div
  className="jumbotron"
  style="100px"
>
  <div
    className="img"
  />
  <div
    className="container"
  >
    <h1
      className="display-3"
    >
      Title
    </h1>
    <hr
      className="my-2"
    />
    <p
      className="lead"
    >
      Subtitle
    </p>
  </div>
</div>
`;

exports[`Jumbotron Component renders consistently 1`] = `
<div
  className="jumbotron"
  style="100px"
>
  <div
    className="img"
  />
  <div
    className="container"
  >
    <h1
      className="display-3"
    >
      Title
    </h1>
    <hr
      className="my-2"
    />
    <p
      className="lead"
    >
      Subtitle
    </p>
  </div>
</div>
`;
